lecture/tests.py

```
@@ -1,3 +1,92 @@
+LECTURE_TITLE = "Django ORM Basics"
+LECTURE_DESC = "An overview of Django's ORM capabilities."
+PDF_TITLE = "Lecture Notes"
+PDF_FILE_NAME = "lecture_notes.pdf"
+VIDEO_TITLE = "Django ORM Walkthrough"
+VIDEO_FILE_NAME = "django_tutorial.mp4"
+VIDEO_CONTENT = b"video content"
+PDF_CONTENT = b"PDF content"
+
+
+class LectureModelTest(TestCase):
+    def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)